app/assets/stylesheets/styles/components/guides/_edit_guides.css.scss

```diff
@@ -60,34 +60,34 @@
 
 
 
-  input[type=text].hidden-edit-field{
+  input[type=text].hidden-edit-field {
     font-size: 2rem;
     height: 3.5rem;
   }
 
-  p.section-text{
+  p.section-text {
     margin-top: 1rem;
   }
 
 
-  .edit-guide-app .box, .show-guide-app .box{
+  .edit-guide-app .box, .show-guide-app .box {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line